tools/README.md

```diff
@@ -1 +1,20 @@
-Command line utilities for processing and converting 3D Tiles tilesets.
\ No newline at end of file
+# 3D Tiles Tools
+
+Command line utilities for processing and converting 3D Tiles tilesets.
+
+## Instructions
+
+Clone this repo and install [Node.js](http://nodejs.org/).  From the root directory of this repo, run:
+```
+npm install
+```
+
+## Tools
+
+### gzipTileset
+
+Detects whether the tileset is compressed or not and does the opposite.
```

Comment:
The other option - that I would be OK with - is that each stage can be a separate command-line program and then the master pipeline command-line program would take the JSON config; however, this is different than, for example, `git`, `gzip`, etc.